### PR TITLE
2.0.8

### PR DESCRIPTION
**Pull request by someone2639**
the j

### CODE REVIEW

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-v2.0.7
+v2.0.9
```
Comment (thecozies) [inline VERSION.txt:1]: the

**Review by thecozies — CHANGES_REQUESTED** [review]

File: VERSION.txt (hunk 1 above)
Comment (gheskett) [inline VERSION.txt:1]: True!

**Comment by gheskett** [comment]
> i'm going to commit sudoku

|   | 3 | 1 | 4 |   |   |   |   |   |
| - | - | - | - | - | - | - | - | - |
|   |   | 8 |   |   |   |   | 3 |   |
|   |   |   |   |   | 8 | 9 |   | 7 |
| 4 | 9 |   | 1 |   |   | 8 |   | 5 |
|   |   | 6 | 5 |   | 9 |   | 7 |   |
|   |   |   |   |   |   | 1 |   |   |
| 3 |   | 4 |   | 1 | 7 |   | 9 |   |
|   | 6 |   |   | 9 |   | 5 |   |   |
| 8 |   |   | 2 |   |   |   |   |   |

**Review by gheskett — APPROVED** [review]
F

**Review by axollyon — APPROVED** [review]
should've been 2.0.∞

**Comment by axollyon** [comment]
wait why tf is your repo still called ultrasm64-extbounds

**Comment by gheskett** [comment]
> image

First 3x3 box has two 1's; try again

**Review by thecozies — APPROVED** [review]

**Comment by Reonu** [comment]
sooooo are we merging this

**Pull request merged**